signer-wasm/lib/session.js

```diff
@@ -0,0 +1,29 @@
+import { DeviceNotSupported, UnknownDevice } from './errors';
+import FilecoinApp from "@zondax/ledger-filecoin"
+
+const DeviceEnum = {
+  LEDGER: 'ledger',
+  TREZOR: 'trezor'
+};
+
+
+// Utility class to hold hardware device session
+class DeviceSession {
+  constructor (device) {
+    this.device = device;
+
+    switch (device) {
+      case DeviceEnum.LEDGER:
+        this.session = new FilecoinApp();
```

Comment:
I was expecting the session to contain the transport.. we can refactor that later..  
However, what is odd here is that the app is initialized without transport.. this won't work

Reply:
Indeed I have missed that. I don't have test for this. I was hoping I could use the emulator.